Xamarin.Essentials/Contacts/Contacts.ios.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Contacts;
+using ContactsUI;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static Task<Contact> PlatformPickContactAsync()
+        {
+            var uiView = Platform.GetCurrentViewController();
+            if (uiView == null)
+                throw new ArgumentNullException($"The View Controller can't be null.");
```

Comment:
Maybe use a more "helpful" message for the developer, like "Unable to find the current view controller." so that at least it makes more sense as to _why_ it crashed.